### PR TITLE
Add script to import ngeo apps

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

**Comment by sbrunner** [comment]
Also remove the scripts and make them on the build

**Comment by sbrunner** [comment]
Finally I completely remove the imported files from the c2cgeoportal repository

Ready to be reviewed

**Comment by sbrunner** [comment]
fix  #1847

File: doc/developer/ngeo.rst [hunk 1]
```diff
@@ -10,184 +10,10 @@ Integrate demo apps in c2cgeoportal scaffold
 Copy Files
 ----------
 
-Copy the following files from ngeo:
+We have an automatic tash for this:
```
Comment (ger-benjamin) [inline doc/developer/ngeo.rst]: tash -> task ?

**Pull request merged**